db/hsuforum_custom_drafts.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<XMLDB PATH="mod/hsuforum/db" VERSION="20211019" COMMENT="XMLDB file for mod hsuforum"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../../../lib/xmldb/xmldb.xsd">
+  <TABLES>
+    <TABLE NAME="hsuforum_custom_drafts" COMMENT="Stores a custom draft version of a post">
+      <FIELDS>
+        <FIELD NAME="id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="true"/>
+        <FIELD NAME="userid" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false"/>
+        <FIELD NAME="forumid" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false"/>
+        <FIELD NAME="discussionid" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false"/>
+        <FIELD NAME="postid" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false"/>
+        <FIELD NAME="text" TYPE="text" NOTNULL="true" SEQUENCE="false"/>
+        <FIELD NAME="timemodified" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
+      </FIELDS>
+      <KEYS>
+        <KEY NAME="primary" TYPE="primary" FIELDS="id"/>
+      </KEYS>
+    </TABLE>
+  </TABLES>
+</XMLDB>
```

Comment:
Motivation for having this here and in the install.xml?

Reply:
Install will have it as a complete entry. Reason for the separate file is so that the entire table definition can be loaded as one xml file in the upgrade script

`$dbman->install_one_table_from_xmldb_file($CFG->dirroot.'/mod/hsuforum/db/hsuforum_custom_drafts.xml', 'hsuforum_custom_drafts');`

Reply:
Suggestion: You can install a single table from install.xml with 
`install_one_table_from_xmldb_file($file, $tablename, $cachestructures = false)`
`install_one_table_from_xmldb_file($CFG->dirroot.'/mod/hsuforum/db/install.xml, 'hsuforum_custom_drafts')`

Reply:
Sure. Going forward we need to decide on a standard, as we have it done different ways across multiple plugins (so like local_getsmarter has it done this way)

Reply:
Cool cool - as discussed since this is working we will use this going forward to avoid duplication.